css/style.css

```diff
@@ -1,22 +1,34 @@
 /* global */
 @import url("https://fonts.googleapis.com/css?family=Roboto");
 
+html {
+  height: 100%;
+}
+
```

Comment:
found out that the page only renders roughly half my screen size.
adding this fixed it.